web/client/api/__tests__/CSW-test.js

```diff
@@ -108,6 +108,30 @@ describe('Test correctness of the CSW APIs', () => {
             }
         });
     });
+
+    it('does not include capabilities when no parsedUrl in getRecords', (done) => {
+        API.getRecords('base/web/client/test-resources/csw/getRecordsNoWMS.xml', 1, 1).then((result) => {
+            try {
+                expect(result).toBeTruthy();
+                expect(result.records[0].capabilities).toNotExist();
+                done();
+            } catch (ex) {
+                done(ex);
+            }
+        });
+    });
+
+    it('includes capabilities when parsedUrl exists in getRecords', (done) => {
+        API.getRecords('base/web/client/test-resources/csw/getRecordsResponseDC.xml', 1, 1).then((result) => {
+            try {
+                expect(result).toBeTruthy();
+                expect(result.records[0].capabilities).toBeTruthy();
```

Comment:
this check is the same for **getRecords update capabilities**
can you do something more specific or check values

in this case we can merge these two unit tests